plugins/snowflake/scripts/classify_issues.py

```
@@ -113,23 +124,67 @@ def build_prompt(batch):
+
+    Prefers google.auth.default() which automatically supports:
+    - Application Default Credentials (ADC)
+    - Workload identity
+    - Service account keys
+    - User credentials via gcloud auth application-default login
+
+    Falls back to gcloud CLI if google-auth library is not available.
+    """
+    if USE_GOOGLE_AUTH:
+        try:
+            credentials, project = google.auth.default()
```

Ruff (0.15.17)

[warning] Unpacked variable `project` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)